subtests/docker_cli/run_cgroups/cgroup_parent.py

```diff
@@ -0,0 +1,103 @@
+"""
+Tests for bz1320302 - odd behavior of --cgroup-parent option
+
+Operational Summary
+----------------------
+
+#. Start container with --cgroup-parent=X, for various values of X
+#. run container command 'tail -n1 /proc/1/cgroup'
+#. Compare to expected result
+"""
+
+
+import os.path
+import re
+from autotest.client import utils
+from dockertest.dockercmd import DockerCmd
+from dockertest.images import DockerImage
+from dockertest.output import OutputGood
+from dockertest.xceptions import DockerTestError, DockerTestFail
+from cgroups_base import cgroups_base
+
+
+class cgroup_parent_base(cgroups_base):
+    def _setup(self, cg_parent, expected_path, expected_stderr):
```

Comment:
pylint might get mad there's no space between class/def.  I'll let it be the judge though :smile: